pages.pt_BR/common/dd.md

```diff
@@ -3,22 +3,22 @@
 > Converte e copia um arquivo.
 > Mais informações: <https://manned.org/dd.1p>.
 
-- Cria um USB drive bootável a partir de um arquivo isohybrid (como uma `archlinux-xxx.iso`) e mostra o progresso:
+- Cria um USB drive inicializável a partir de um arquivo isohybrid (tal como `archlinux-xxx.iso`) e mostra o progresso:
 
-`dd if={{caminho/para/arquivo.iso}} of={{/dev/usb_drive}} status=progress`
+`dd if={{caminho/para/arquivo.iso}} of={{/dev/drive_usb}} status=progress`
 
-- Clona um drive para outro drive com 4 MiB block e ignora erros:
+- Clona um drive para outro drive com bloco de 4 MiB e descarta escritas antes que o comando termine:
 
-`dd bs=4194304 conv=noerror if={{/dev/drive_fonte}} of={{/dev/drive_destino}}`
+`dd bs=4194304 conv=fsync if={{/dev/drive_fonte}} of={{/dev/drive_destino}}`
```

Comment:
What do you think about origem and destino, instead of fonte e destino? I was thinking both are corrects and understandable, but I feel 'origem' looks better.

Reply:
Agreed, `origem` sounds better IMO as well.